### PR TITLE
E1724. Remove duplicated code in feature tests

### PR DESCRIPTION
**Pull request by ylu36**

### CODE REVIEW

File: spec/features/scheduled_task_spec.rb [hunk 1]
```diff
@@ -1,132 +1,86 @@
...
-    due_at1 = Time.parse(due_at.to_s(:db))
-    curr_time = DateTime.now.in_time_zone.to_s(:db)
-    curr_time = Time.parse(curr_time)
-    time_in_min = ((due_at1 - curr_time).to_i / 60) * 60
-    Delayed::Job.delete_all
-    expect(Delayed::Job.count).to eq(0)
-
-    Delayed::Job.enqueue(payload_object: ScheduledTask.new(id, "submission", due_at), priority: 1, run_at: time_in_min)
-
+    # id = 2
```
Comment (Winbobob) [inline spec/features/scheduled_task_spec.rb]: You do not need to commit these two lines, delete them.

Comment (Winbobob) [inline spec/features/scheduled_task_spec.rb]: After finishing doing this, please let me know and I will approve other code climate issues.

Comment (ylu36) [inline spec/features/scheduled_task_spec.rb]: what two lines?

Comment (Winbobob) [inline spec/features/scheduled_task_spec.rb]:
```
# id = 2
# @name = "user"
```

Comment (ylu36) [inline spec/features/scheduled_task_spec.rb]: i updated it

Comment (Winbobob) [inline spec/features/scheduled_task_spec.rb]: Please check the whole file `spec/features/delayed_mailer_spec.rb` and remove all of them.

Comment (ylu36) [inline spec/features/scheduled_task_spec.rb]: should be fixed

File: spec/features/assignment_creation_spec.rb [hunk 2]
```diff
@@ -588,29 +557,6 @@ def fill_assignment_form
       fill_in 'late_policy_max_penalty', with: 2
       click_button 'Create'
     end
-
-    # able to set deadlines for a single round of reviews
-    it "set the deadline for an assignment review" do
```
Comment (Winbobob) [inline spec/features/assignment_creation_spec.rb]: Why did you delete this test?

File: spec/features/questionnaire_spec.rb [hunk 3]
```diff
@@ -89,303 +64,85 @@ def load_questionnaire
     click_button "Create"
   end
 
-  describe "Create a review question" do
```
Comment (Winbobob) [inline spec/features/questionnaire_spec.rb]: Why did you delete these tests?

File: spec/features/questionnaire_spec.rb [hunk 4]
```diff
@@ -89,303 +64,85 @@ def load_questionnaire
...
-      click_on('Remove')
-      expect(page).to have_content('You have successfully deleted the question!')
-    end
-
-    it "is able to delete a TextArea question" do
-      load_question 'TextArea'
-
-      click_on('Remove')
-      expect(page).to have_content('You have successfully deleted the question!')
+  describe "Create a review question" do
```
Comment (Winbobob) [inline spec/features/questionnaire_spec.rb]: LGTM 👍

File: spec/features/questionnaire_spec.rb [hunk 5]
```diff
@@ -89,303 +64,85 @@ def load_questionnaire
...
+      check_deleted_question
     end
+  end
 
-    it "is able to delete a SectionHeader question" do
-      load_question 'SectionHeader'
-
-      click_on('Remove')
-      expect(page).to have_content('You have successfully deleted the question!')
+  describe "Edit and delete a question" do
```
Comment (Winbobob) [inline spec/features/questionnaire_spec.rb]: LGTM 👍

File: spec/features/assignment_creation_spec.rb (hunk 2 above)
Comment (Winbobob) [inline spec/features/assignment_creation_spec.rb]: I just afraid the code may not that readable for later developers. But you did a great job! And there may be a tradeoff between readable and duplicate code.
